### DISCUSSION TITLE
Vite instead of @vue/cli-service ?

### DISCUSSION THREAD

**Discussion by mariusa** [GraphiteEditor/Graphite#956] [Ideas/Feedback]
Hi, 

Even though the project uses Vue 3, I was surprised @vue/cli-service is used instead of vite https://vitejs.dev/ , which is significantly faster.

Any reasons for that?

Thanks

**Comment by Keavon** [comment]
Regardless of our frontend framework, we have to use webpack for the moment because wasm-pack and other build dependencies require it. Once we finish the port to Svelte, it will be useful to begin investigating whether we can get wasm-pack to work in another bundler (Vite, Rollup, or others) or replace wasm-pack with something like Trunk. Anyone who wants to help with this would be welcome to do so!

**Comment by Keavon** [comment]
This is done now.
